crates/codegen/parser/runtime/src/cst.rs

```diff
@@ -22,6 +22,13 @@ pub struct TokenNode {
 pub enum Node {
     Rule(Rc<RuleNode>),
     Token(Rc<TokenNode>),
+    Error(Rc<ErrorNode>),
```

Comment:
I wonder if the new `ErrorNode` type replaces `TokenKind::SKIPPED`. Should we remove it in that case to avoid confusion?